components/SignInOrJoinFree.js

```diff
@@ -248,7 +248,11 @@ class SignInOrJoinFree extends React.Component {
                       onSubmit={handleSubmit}
                       data-cy={useRecoveryCodes ? null : 'signin-two-factor-auth-button'}
                     >
-                      <FormattedMessage id="VerifyButton" defaultMessage="Verify" />
+                      {useRecoveryCodes ? (
+                        <FormattedMessage id="login.twoFactorAuth.redefine" defaultMessage="Redefine 2FA" />
```

Comment:
I know it was in the spec, but I would prefer `Reset 2FA` instead of `Redefine 2FA`.